tools/nnpackage_tool/model2nnpkg/model2nnpkg

```diff
@@ -0,0 +1,143 @@
+            config_path = os.path.basename(args.config[i])
+            if not os.path.isfile(config_path):
+                raise Exception(f'error: {config_path} does not exist.')
+
+
+def _get_args():
+    parser = argparse.ArgumentParser(
+        description='Convert model files (tflite, circle or tvn) to nnpkg.',
+        usage=''' %(prog)s [options]
+  Examples:
+      %(prog)s -m add.tflite                           => create nnpkg "add" in current directory
+      %(prog)s -o out -m add.tflite                    => create nnpkg "add" in out/
+      %(prog)s -o out -p addpkg -m add.tflite          => create nnpkg "addpkg" in out/
+      %(prog)s -c add.cfg -m add.tflite                => create nnpkg "add" with add.cfg
+      %(prog)s -o out -p addpkg -m a1.tflite a2.tflite => create nnpkg "addpkg" with models a1.tflite and a2.tflite in out/
+  ''')
+    parser.add_argument(
+        '-o',
+        '--outdir',
+        type=str,
+        default=os.getcwd(),
+        metavar='output_directory',
+        help='set nnpkg output directory')
+    parser.add_argument(
+        '-p',
+        '--nnpkg-name',
+        type=str,
+        metavar='nnpkg_name',
+        help='set nnpkg output name (default=[1st modelfile name])')
+    parser.add_argument(
+        '-c',
+        '--config',
+        type=str,
+        nargs='+',
+        default='',
+        metavar='conf',
+        help='provide configuration files')
+    parser.add_argument(
+        '-m',
+        '--models',
+        type=str,
+        nargs='+',
+        metavar='model',
+        help='provide model files')
+
+    args = parser.parse_args()
+
+    _verify_args(args)
+
+    if not args.nnpkg_name:
+        first_model_name = os.path.basename(args.models[0]).rsplit('.', 1)[0]
+        args.nnpkg_name = first_model_name
+
+    args.prog = parser.prog
+
+    return args
+
+
+def _generate_manifest(args):
+    config_list = ''
+    if args.config:
+        config_list = [os.path.basename(e) for e in args.config]
+    models_list = [os.path.basename(e) for e in args.models]
+    types_list = [os.path.basename(e).rsplit('.', 1)[1] for e in args.models]
+
+    manifest = {}
+    manifest["major-version"] = "1"
+    manifest["minor-version"] = "2"
+    manifest["patch-version"] = "0"
+    manifest["configs"] = config_list
+    manifest["models"] = models_list
+    manifest["model-types"] = types_list
+
+    return manifest
+
+
+def main():
+    # parse arguments
+    args = _get_args()
+
+    print(f'{args.prog}: Generating nnpkg {args.nnpkg_name} in {args.outdir}')
+    # mkdir nnpkg directory
+    nnpkg_path = os.path.join(args.outdir, args.nnpkg_name)
+    os.makedirs(os.path.join(nnpkg_path, 'metadata'), exist_ok=True)
+
+    # dump manifest file
+    manifest = _generate_manifest(args)
+    manifest_path = os.path.join(nnpkg_path, 'metadata', 'MANIFEST')
+    with open(manifest_path, "w") as json_file:
+        json.dump(manifest, json_file, sort_keys=True, indent=2)
```

Comment:
@ragmani I prefer not to sort by keys. I would like to keep original order w/o changing order.

Reply:
OK, I will change it.